includes/admin.form.inc

```diff
@@ -97,6 +103,18 @@ function islandora_repository_admin(array $form, array &$form_state) {
           '#description' => t('Larger sites may experience a notable performance improvement when disabled due to how breadcrumbs are constructed.'),
           '#default_value' => variable_get('islandora_render_drupal_breadcrumbs', TRUE),
         ),
+        'islandora_breadcrumbs_backends' => array(
+          '#type' => 'radios',
+          '#title' => t('Breadcrumb generation'),
+          '#description' => t('How breadcrumbs for Islandora objects are generated for display.'),
+          '#default_value' => variable_get('islandora_breadcrumbs_backends', ISLANDORA_BREADCRUMB_LEGACY_BACKEND),
```

Comment:
Usability: Might want to check whether the option that variable_get returns is in $breadcrumb_backend_options. If I enable Awesome Breadcrumbs and select it, then disable the Awesome Breadcrumbs, I get a weird form with an unselected radio button and it's hard to see why my form won't submit.